spec/models/provider_spec.rb

```diff
@@ -140,4 +140,16 @@
 
     its(:unassigned_site_codes) { should eq(expected_unassigned_codes) }
   end
+
+  describe "#can_add_more_sites?" do
+    it "should return true when provider has less sites than max allowed" do
+      provider = create(:provider, site_count: 0)
+      expect(provider.can_add_more_sites?).to eq(true)
+    end
+
+    it "should return false when provider has the max sites allowed" do
+      provider = create(:provider, site_count: Site::POSSIBLE_CODES.size)
+      expect(provider.can_add_more_sites?).to eq(false)
+    end
+  end
```

Comment:
I'd have a slight preference for the following style which is what we've been doing in other places:

```ruby
  describe "#can_add_more_sites?" do
    context "when provider has less sites than max allowed" do
      subject { create(:provider, site_count: 0) }
      its(:can_add_more_sites?) { should be_truthy }
    end

    context "when provider has the max sites allowed" do
      subject { create(:provider, site_count: 37) }
      its(:can_add_more_sites?) { should be_falsey }
    end
  end
```

@misaka et al, thoughts?